en/spend-bitcoin.html

```diff
@@ -0,0 +1,20 @@
+---
```

Comment:
This is the only request I'm not exactly sure how to handle... As [I mentioned in a comment above](https://github.com/bitcoin-dot-org/bitcoin.org/pull/1584#issuecomment-299935193), I was not able to get Jekyll to respond (to compile, generate, etc.) to files I placed under `_templates` -- which was where I initially attempted to place the new page. @Microbit seemed to [have the same issue](https://github.com/bitcoin-dot-org/bitcoin.org/pull/1584#issuecomment-300843311). Any thoughts/hints?